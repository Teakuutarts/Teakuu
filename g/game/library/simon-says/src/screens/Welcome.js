import React from "react";
import { Flex } from "reflexbox";

import Title from "../components/Title";
import Shell from "./Shell";
import { ButtonLink } from "../components/Buttons";

const Welcome = () => (
  <Shell>
    <Title value="REACT SIMON" />
    <Flex justify="center">
      <ButtonLink to="/board">Play</ButtonLink>
    </Flex>
  </Shell>
);

export default Welcome;
