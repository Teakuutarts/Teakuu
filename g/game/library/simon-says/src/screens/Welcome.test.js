import React, { Component } from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Welcome from './Welcome';

it('renders Welcome correctly', () => {
  const wrapper = shallow(
    <Welcome />
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});

