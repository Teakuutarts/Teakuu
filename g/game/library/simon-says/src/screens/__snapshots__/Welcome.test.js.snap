exports[`test renders Welcome correctly 1`] = `
<Connect(Shell)>
  <Title
    value="REACT SIMON" />
  <Flex
    justify="center">
    <Styled(Link)
      to="/board/">
      Play
    </Styled(Link)>
  </Flex>
</Connect(Shell)>
`;
